src/tool_source/github.rs

```diff
@@ -18,20 +20,25 @@ const APP_NAME: &str = "LPGhatguy/aftman";
 
 pub struct GitHubSource {
     client: Client,
+    token: Option<String>,
 }
 
 impl GitHubSource {
-    pub fn new() -> Self {
+    pub fn new(home: &Home) -> Self {
+        let token = AuthManifest::load(home).ok();
```

Comment:
Should we load this file in another place? We shouldn't be suppressing errors here with `ok()`. If we want to load it here, we can change `GitHubSource::new` to return an `anyhow::Result<Self>` instead.

Reply:
I've attempted to fix this, but will need some help with borrowing as I'm still new to rust!